hw/top_earlgrey/data/ip/chip_sysrst_ctrl_testplan.hjson

```diff
@@ -5,77 +5,191 @@
   testpoints: [
     {
-      name: chip_sw_sysrst_ctrl_combo_reset
-      desc: '''Verify that sysrst_ctrl issue reset on combo detection
+      name: chip_sw_sysrst_ctrl_reset
+      desc: '''Verify the SYSRST ctrl can reset the chip from normal state.
 
-            - Initialize sysrst_ctrl with combo detect and output of combo detect as rst_req assertion.
-            - Wait for a delay greater than combo detect timer
-            - Check if there is a reset and if reset source is sysrst_ctrl
-            - Check if sysrst_ctrl.WKUP_STATUS.wakeup_sts is set
+            - Read the reset cause register in rstmgr to confirm we are in POR reset phase.
+            - Program one of the com_sel_ctl_* CSRs to choose a set of inputs to be detected as
+              the chip reset signal.
+            - Program the associated detection timer.
+            - Program the detection outcome CSR's (com_out_ctl) chip reset bit to 1.
+            - After some time, set the SYSRST ctrl inputs at the chip IOs to the programmed
+              combination for the duration of the detection timer.
+            - The pwrmgr will power cycle the chip once it receives the chip reset input.
+            - Check that ec_rst_l and flash_wp_l (on pads IOR8 and IOR9) are asserted right after
+              the pwrmgr has power cycled the system.
+            - Read the reset cause register after boot up to confirm peripheral reset phase.
+            - Read the pwrmgr reset status register to confirm chip reset.
+            - Read the com_sel_ctl_* CSR in SYSRST ctrl we programmed earlier - it should have been
+              reset.
             '''
       features: ["SYSRST_CTRL.COMBO_DETECT"]
-      stage: V3
+      stage: V2
       si_stage: SV3
       lc_states: ["DEV", "PROD", "PROD_END", "RMA"]
       tests: ["chip_sw_pwrmgr_sysrst_ctrl_reset", "chip_sw_sysrst_ctrl_reset"]
     }
     {
-      name: chip_sw_sysrst_ctrl_input
-      desc: '''Verify that SW can access the raw value of input signal via sysrst_ctrl
-
-            - Initialize sysrst_ctrl
-            - set the following input signals with a random value
-              - pwrb_in_i
-              - key2_in_i
-              - key1_in_i
-              - key0_in_i
-              - ac_present_i
-              - ec_rst_l_i
-              - flash_wp_l_i
-            - Wait for values to synchronize
-            - Read sysrst_ctrl.PIN_IN_VALUE and check if it matches the values being driven on inputs
-            - Repeat the sequence for different combinations of input signals
+      name: chip_sw_sysrst_ctrl_inputs
+      desc: '''Verify that the SYSRST ctrl input pin values can be read.
 
+            - Drive a known value on ac_reset, ec_rst_l, flash_wp_l, pwrb, lid_open and key* pins at
+              the chip inputs.
+            - Read the pin_in_value CSR to check for correctness.
             '''
       features: ["SYSRST_CTRL.PIN_INPUT_VALUE_ACCESS"]
-      stage: V3
+      stage: V2
```

Comment:
This was presumably marked as V3 because it wasn't a core feature. We can promote it to V2 if you think we need to do so, but that should probably be a separate change (even if it's just a separate commit in the same PR).

Reply:
I don't really understand what `V2` versus `V3` means. It seems that the tests in `sysrst_testplan` are all V3 and are duplicated of tests in `chip_testplan` which are V2 (and the chip testplan contains more tests). Should I keep all tests that are duplicate as `V3`?

Reply:
Er, this is the same problem again and I'd got confused about which test came from which file! Looking again, this test was always marked V2. Sorry for the noise.